runtime/common/src/constants.rs

```diff
@@ -226,6 +189,7 @@ pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"py/trsry");
 
 // https://wiki.polkadot.network/docs/learn-treasury
 // https://paritytech.github.io/substrate/master/pallet_treasury/pallet/trait.Config.html
+// Needs parameter_types! for the Permill and PalletId
```

Comment:
I added a simple comment each time we use `parameter_types!` as to why we still have to use it